h/static/styles/partials-v2/_search.scss

```diff
@@ -100,9 +100,6 @@
 }
 
 .search-result-bucket__content {
-  display: flex;
-  flex-direction: column;
-  flex-grow: 1;
```

Comment:
`search-result-bucket__content` only contains one element (`search-result-bucket__annotation-cards-container`) so this flex layout seems to be unnecessary. I removed it (in a separate commit).